subsys/bluetooth/audio/codec.c

```diff
@@ -42,6 +112,15 @@ static bool parse_cb(struct bt_data *data, void *user_data)
 
 #if CONFIG_BT_AUDIO_CODEC_CFG_MAX_DATA_SIZE > 0
 
+static void init_net_buf_simple_from_codec_cfg(struct net_buf_simple *buf,
+					       struct bt_audio_codec_cfg *codec_cfg)
```

Comment:
```suggestion
static void init_net_buf_simple_from_codec_cfg(struct net_buf_simple *buf,
					       const struct bt_audio_codec_cfg *codec_cfg)
```

Reply:
Can't do `const` as `codec_cfg->data` cannot be stored in a non-const pointer like `buf->data`